packs/boons-and-curses/sarenrae-major-curse.json

```
@@ -24,7 +24,22 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Gods & Magic"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "AdjustStrike",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "nor": [
+                            "target:trait:fiend",
+                            "target:mode:undead"
+                        ]
+                    }
+                ],
+                "property": "weapon-traits",
+                "value": "nonlethal"
+            }
+        ],
```

```suggestion
        "rules": [],
```
The nonlethal trait isn't the same as nonlethal damage, so best to leave that as just a note maybe. We can also use an ephemeral here